files/build_templates/sonic_debian_extension.j2

```diff
@@ -1028,6 +1028,8 @@ sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable nv-syncd-shared
 
 # Install minicom package
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install minicom
+
+sudo LANG=C chroot $FILESYSTEM_ROOT systemctl disable rshim.service
```

Comment:
Is it possible to have the service disabled by default when building it? I mean this line is not needed on other vendor's images, right?

Reply:
I'm not aware of the way to build disabled by default. Since we build/install it only for Mellanox, this line is only needed for Mellanox.

Reply:
@Yakiv-Huryk why is this line of change needed? seems this service will only be needed for Mellanox so why do we need to explicity disable?

Reply:
@Yakiv-Huryk If `--no-enable` is passed into `dh_installsystemd` when the deb package is built, then it will get installed as disabled by default, and won't get automatically enabled.